server/server/src/lib.rs

```diff
@@ -39,7 +39,7 @@ pub mod sync;
 pub mod test_utils;
 
 // Only import discovery for non android features (otherwise build for android targets would fail due to local-ip-address)
-#[cfg(not(feature = "android"))]
+#[cfg(not(target_os = "android"))]
```

Comment:
the feature is always enabled, and I couldn't see how to make a workspace member conditional. This seemed the simplest option - have tested by compiling on the build server (build.msupply.org) and by running the server locally.

Reply:
Why is the android feature always enabled?

Reply:
Right now because of this: https://github.com/openmsupply/open-msupply/blob/ced8e47a08c43957d04e0718a57a6a993fe006c4/server/android/Cargo.toml#L14

Previous because of https://github.com/openmsupply/open-msupply/blob/b5e211a88339651aa83ee8d75a816a571ce811df/server/android/Cargo.toml#L25

Workspaces unify features. If we build for explicit targets this wouldn't be an issue btw `cargo build --bin remote_server`